docs/api/identity-service.md

```diff
@@ -162,10 +162,6 @@ The `type` query parameter specifies the identity type to return. Accepted value
         "deviceId": "device01",
         "moduleId": "module01",
         "genId": "12345",
-        "auth": {
```

Comment:
AFAIK, this key is still in the response.

Reply:
It's absent in the response I received when testing this API and AFAIK it has never been included per this [old issue](https://github.com/Azure/iot-identity-service/issues/56).  I don't believe it's required so to resolve the issue I'm updating the docs to align with the implementation.

Reply:
The code to include it exists though. I think it will be present in certain cases (though I haven't tried it).